data/campaigns/Wings_of_Victory/_main.cfg

```diff
@@ -0,0 +1,70 @@
+#textdomain wesnoth-wov
+[textdomain]
+    name="wesnoth-wov"
+[/textdomain]
+
+# wmlscope: set export=no
+[campaign]
+    id=Wings_of_Victory
+    name= _ "Wings of Victory"
+    abbrev= _ "WoV"
+    icon="units/drakes/inferno-fly-2-upstroke.png~CROP(0,0,72,72)~RC(magenta>red)"
+    rank=120
+    start_year="5 YW"
+    end_year="6 YW"
+    define=CAMPAIGN_WINGS_OF_VICTORY
+    first_scenario="01_The_Hunt"
+    image="data/core/images/portraits/drakes/inferno.png~SCALE(350,350)"
+    description= _ "Seize, conquer, dominate, expand. That was ever the Way of the Drakes. But the Brothers of the Spiral Path have seen a vision: Irdya is made like an egg, and the day will come when there is no more world to conquer - and what of the Way then? There was a drake named Galun on whom fell the burden of finding an answer...
+
+" + _ "(Intermediate level, 11 scenarios.)" + "
+" + _ "Version:" + " " + "{./dist/version}" + "  " + "https://r.wesnoth.org/t49140"
+
+    {CAMPAIGN_DIFFICULTY EASY   "units/drakes/burner-fly-1.png~CROP(0,0,72,72)~RC(magenta>red)"        ( _ "Fledgling") ( _ "Normal")}
+    {CAMPAIGN_DIFFICULTY NORMAL "units/drakes/fire-fly-2-upstroke.png~CROP(0,0,72,72)~RC(magenta>red)" ( _ "Aspirant")  ( _ "Challenging")} {DEFAULT_DIFFICULTY}
+    {CAMPAIGN_DIFFICULTY HARD   "units/drakes/inferno-fly-5.png~CROP(0,0,72,72)~RC(magenta>red)"       ( _ "Dominant")  ( _ "Difficult")}
+
+    [about]
+        title= _ "Authors"
+        [entry]
+            name="Fabi/Fendrin"
+        [/entry]
+        [entry]
+            name="Eric S. Raymond (esr)"
+        [/entry]
+        [entry]
+            name = "Jeffrey 'Sigurd' Westcoat (SigurdFireDragon)"
+        [/entry]
+    [/about]
+    [about]
+        title = _ "Alpha Testing and Proofreading"
+        [entry]
+            name = "josteph"
+        [/entry]
+    [/about]
+    [about]
+        title = _ "Campaign Maintenance"
+        [entry]
+            name = "Jeffrey 'Sigurd' Westcoat (SigurdFireDragon)"
+            comment= "current maintainer"
+        [/entry]
+    [/about]
+[/campaign]
+
+#ifdef CAMPAIGN_WINGS_OF_VICTORY
+[binary_path]
+    path=data/campaigns/Wings_of_Victory
+[/binary_path]
+
+{./utils}
+{./scenarios}
+[+units]
+    {./units}
+[/units]
+#endif
+
+# TODO: use em dashes & unicode quote? Pango doesn't seem to like it on my system,
```

Comment:
Could this be because you haven't told your editor to save as UTF-8?

Reply:
Will give this a try, thanks.